pkg/virt-handler/vm.go

```diff
@@ -2010,7 +2010,14 @@ func (d *VirtualMachineController) heartBeat(interval time.Duration, stopCh chan
 				log.DefaultLogger().Reason(err).Errorf("Can't determine date")
 				return
 			}
-			data := []byte(fmt.Sprintf(`{"metadata": { "labels": {"%s": "true"}, "annotations": {"%s": %s}}}`, v1.NodeSchedulable, v1.VirtHandlerHeartbeat, string(now)))
+
+			kubevirtSchedulable := "true"
+			sp := device_manager.SocketPath(device_manager.KVMName)
```

Comment:
Sorry for the late change request, but I think this is a little bit too unclear. We should establish a clear check.

How about explicitly probing our internal device plugins if the successfully managed to register to the kubelet?

Reply:
@rmohr would you mind relooking over this PR? I have updated it to have a clear check. If it seems like the correct way to go ill add tests for it.